src/backend/metal/src/lib.rs

```diff
@@ -958,6 +959,11 @@ impl PrivateCapabilities {
             max_texture_layers: 2048,
             max_fragment_input_components: if os_is_mac { 128 } else { 60 },
             sample_count_mask,
+            supports_debug_markers: if os_is_mac {
```

Comment:
could we rewrite that to be consistent with the other checks, which are using `supports_any`?

Reply:
Not with confidence -- i'm not clear how `MTLFeatureSet` works: if i test for [macOS_GPUFamily1_v2](https://developer.apple.com/documentation/metal/mtlfeatureset/macos_gpufamily1_v2), will that check also be successful for later OS versions, or do i need to explicitly list them too?

Reply:
IIRC, the versions are backwards compatible, but GPU families are not. So you'd list the pairs of (OS, GPU Family) -> minimal supported version, if any.

Reply:
Explicitly listing the GPU families where the function exists seems like the wrong thing to do, because when future ones come out the check will incorrectly think they don't support the function?

Reply:
I agree this is a tricky thing, but I don't see a better alternative. When a new GPU family comes out (and they do come out), we don't know if a particular feature is supported. So I think it's safest to assume no feature until we inspect the updated spec and update these listings with the new family.